packages/cfpb-typography/src/atoms/tagline.less

```diff
@@ -0,0 +1,52 @@
+.a-tagline {
+    display: grid;
+    grid-template-columns: 22px 1fr;
+    grid-column-gap: 10px;
+
+    // This'll target the USA flag.
+    & > *:first-child {
```

Comment:
Is this wildcard preferred over specifically targeting the flag element? Is this because there *could* be a different non-flag element that comes first?

Reply:
I was just making it generic that way, but I guess that's highly unlikely and it would probably render ever so quicker without a wildcard.

Fixed in https://github.com/cfpb/design-system/pull/1301/commits/4f615970bd6ccf87bd1bf3f5983b135b60cdc131